subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/BuildVerdictPluginExtension.kt

```diff
@@ -0,0 +1,10 @@
+package com.avito.android.build_verdict
+
+import org.gradle.api.file.Directory
+import org.gradle.api.file.ProjectLayout
+
+abstract class BuildVerdictPluginExtension(
+    projectLayout: ProjectLayout
+) {
+    val buildVerdictOutputDir: Directory = projectLayout.projectDirectory.dir("outputs/build-verdict")
```

Comment:
extension, but with val? why?

Reply:
Change. I thought that the Directory is Property